apps/domain/src/main/core/infrastructure/tf.py

```diff
@@ -62,13 +63,22 @@ def destroy(self):
         )
 
     def install_plugins(self, dir):
-        if not os.path.exists(os.path.join(dir, "registry.terraform.io")):
+        zip_file_suffix = str(platform.system()).lower()
+        if zip_file_suffix == "":
+            raise Exception("Can not determine operating system")
+        elif zip_file_suffix == "java":
+            raise Exception("Terraform does not support this operating system")
+
+        file_dir = os.path.join(
+            dir, f"registry.terraform.io/hashicorp/aws/3.30.0/{zip_file_suffix}_amd64/"
+        )
+        if not os.path.exists(file_dir):
             return subprocess.run(
-                """
+                f"""
                 echo "Install terraform plugins"
-                mkdir -p "registry.terraform.io/hashicorp/aws/3.30.0/linux_amd64/"
-                wget https://releases.hashicorp.com/terraform-provider-aws/3.30.0/terraform-provider-aws_3.30.0_linux_amd64.zip
-                unzip terraform-provider-aws_3.30.0_linux_amd64.zip -d "registry.terraform.io/hashicorp/aws/3.30.0/linux_amd64/"
+                mkdir -p "registry.terraform.io/hashicorp/aws/3.30.0/{zip_file_suffix}_amd64/"
+                wget https://releases.hashicorp.com/terraform-provider-aws/3.30.0/terraform-provider-aws_3.30.0_{zip_file_suffix}_amd64.zip
+                unzip terraform-provider-aws_3.30.0_linux_amd64.zip -d "registry.terraform.io/hashicorp/aws/3.30.0/{zip_file_suffix}_amd64/"
```

Comment:
unzip command is missing `{zip_file_suffix}` .
`unzip:  cannot find or open terraform-provider-aws_3.30.0_linux_amd64.zip, terraform-provider-aws_3.30.0_linux_amd64.zip.zip or terraform-provider-aws_3.30.0_linux_amd64.zip.ZIP.`